docs/index.rst

```diff
@@ -1,23 +1,108 @@
-.. pluggy documentation master file, created by
```

Comment:
IMHO our index should only contain an index that points to other documentation... it helps to structure the documentation better. I suggest an index which looks (loosely) like:

* Introduction
* Specs/implementations
* First example
* Managing
* Plugin Registry

Would be nice.

What do others think?